qa/tasks/cephfs/test_volumes.py

```diff
@@ -952,6 +952,50 @@ def test_subvolume_create_idempotence_resize(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_create_idempotence_mode(self):
+        # create subvolume
+        subvolume = self._generate_random_subvolume_name()
+        self._fs_cmd("subvolume", "create", self.volname, subvolume)
+
+        # get subvolume metadata (Expecting default mode 16877 means octal mode 40755)
+        subvol_info_before = json.loads(self._get_subvolume_info(self.volname, subvolume))
```

Comment:
Why not https://github.com/ceph/ceph/blob/cb5d80bc4ff0145c19f869612e9a8de7e50c5690/qa/tasks/cephfs/test_volumes.py#L1089 that is easier to follow? You wouldn't need a separate comment this way

Reply:
Done.